crabs/tracker/utils/io.py

```diff
@@ -29,11 +31,15 @@ def prep_csv_writer(output_dir: str, video_file_root: str):
     Tuple
         A tuple containing the CSV writer, the CSV file object, and the tracking output directory path.
     """
-
-    timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
-    tracking_output_dir = Path(output_dir + f"_{timestamp}") / video_file_root
-    # Create the subdirectory for the specific video file root
-    tracking_output_dir.mkdir(parents=True, exist_ok=True)
+    if run_on_video_dir:
```

Comment:
why do we need two separate cases here? Is it because if we run it on a directory there is no `video_file_root`?

Even if we run inference on a directory with videos, it would be nice that the output is timestamped so that we don't overwrite things right?

Maybe we can replace `video_file_root` with the name of the input directory if a directory of videos is passed.

Reply:
So, I added this because otherwise we will created 200+ folders in the parent directory. Each video will have tracking_output_timestamp as the creating. of the timestamp is here. Hence I made another loop in the bas script itself.
What I tried to do here is to make one timestamp for this run, within the tracking_output_timestamp we will have folder for each videos. I think that makes more sense and easy to look as well.

Reply:
ah ok that sounds good! thanks!